p2p/client.go

```diff
@@ -221,13 +229,14 @@ func (c *Client) SaveBlock(ctx context.Context, height uint64, blockBytes []byte
 }
 
 // RemoveBlocks is used to prune blocks from the block sync datastore.
-func (c *Client) RemoveBlocks(ctx context.Context, from, to uint64) error {
-	if from <= 0 {
-		return fmt.Errorf("from height must be greater than 0: %w", gerrc.ErrInvalidArgument)
-	}
+func (c *Client) RemoveBlocks(ctx context.Context, to uint64) (uint64, error) {
+	prunedBlocks := uint64(0)
 
-	if to <= from {
-		return fmt.Errorf("to height must be greater than from height: to: %d: from: %d: %w", to, from, gerrc.ErrInvalidArgument)
+	from, err := c.store.LoadBlockSyncBaseHeight()
```

Comment:
same as my previous. from will be set to `0` which will cause very high iteration.